php/src/Container/WorkingState.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace AIO\Container;
+
+enum WorkingState: string {
```

Comment:
@docjyJ I just discussed this with Richard and please rename it to `ContainerState`. Otherwise it looks fine :)

Reply:
Do I rename the state version?

Reply:
you mean versionstate? No, that sounds fine

Reply:
```suggestion
enum ContainerState: string {
```

Reply:
> you mean versionstate?

Yes sorry, I wrote that quickly on my phone.